operations/mimir/autoscaling.libsonnet

```diff
@@ -321,6 +342,7 @@
 
           // Threshold is expected to be a string
           threshold: std.toString(std.floor(cpuToMilliCPUInt(cpu_requests) * cpu_target_utilization)),
+          ignore_null_values: false,
```

Comment:
Can we add a comment explaining what ignore_null_values does please?

We should also link out to the keda docs. From memory it's a little counter intuitive.

Reply:
We document that here as well: https://github.com/grafana/mimir/blob/dea203b54f92b46f986d0a81248461f4943dccb1/operations/mimir/autoscaling.libsonnet#L140-L145